meta/meta-node/src/rpc.rs

```diff
@@ -27,27 +28,37 @@ pub struct FullDeps<C, P> {
 }
 
 // Instantiate all full RPC extensions.
-pub fn create_full<C, P>(deps: FullDeps<C, P>) -> jsonrpc_core::IoHandler<sc_rpc::Metadata>
+pub fn create_full<C, P>(
```

Comment:
@canonbrother what was the reason again we changed from `IoHandler` ?🤔
even though substrate bin node use  `Result<RpcModule<()>, Box<dyn std::error::Error + Send + Sync>>`, and that works fine, many other projects use `jsonrpc_core::IoHandler`🤔

Reply:
yea.. here is it:
old:
```diff
-- -> jsonrpc_core::IoHandler<sc_rpc::Metadata> // <-- `sc_rpc::Metadata` is removed on newer deps
++ -> Result<RpcModule<()>, Box<dyn std::error::Error + Send + Sync>> // so using RpcModule by referring latest substrate node template
```

Reply:
This is good. my local had an old code.